backend/capellacollab/config/config_schema.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
We have to mark required properties with the `required` flag. Otherwise, it doesn't complain about missing properties.